server/package.json

```diff
@@ -76,6 +77,7 @@
     "pg": "^8.2.1",
     "pg-cursor": "^2.2.1",
     "pino": "^6.3.2",
+    "prettier": "^2.0.5",
```

Comment:
This shouldn't be necessary if the root package.json was installed, but I think this is fine. I think it might be good to move each dependency into each respective project (server & client)

Reply:
I agree you. I found in my node/npm environment I shoud install the prettier package in the server project, and then it can run correctly, or it's failed.